### PR TITLE
Add GRPC_CONTEXT_TRAFFIC in grpc_context_index

### PR DESCRIPTION
**Pull request by y-zeng**
Add GRPC_CONTEXT_TRAFFIC in grpc_context_index

---
Tests are failing with known issues:
Basic Tests Windows: #9088

### CODE REVIEW

File: src/core/lib/channel/context.h [hunk 1]
```diff
@@ -47,6 +47,9 @@ typedef enum {
   /// Value is a \a census_context.
   GRPC_CONTEXT_TRACING,
 
+  /// Reserved for traffic_classs_context.
```
Comment (dgquintas) [inline src/core/lib/channel/context.h]: classssss :)

**Comment by dgquintas** [comment]
Remember to squash the commits

**Comment by y-zeng** [comment]
Squashed

**Comment by dgquintas** [comment]
Remember also to add the failing tests to the description.

**Comment by y-zeng** [comment]
Thanks for the reminder! I'm waiting for the results of mac tests.

**Pull request merged**